src/main/resources/crafter/engine/mode/preview/services-context.xml

```diff
@@ -69,15 +69,17 @@
         <property name="groovyClassesPath" value="${crafter.engine.site.default.classes.groovy.path}"/>
         <property name="cacheOn" value="false"/>
         <property name="urlTransformationEngine" ref="crafter.urlTransformationEngine"/>
-        <property name="overlayCallback" ref="crafter.authoringOverlayCallback"/>
```

Comment:
I think we need to update the Freemarker templates because of this right?

Reply:
I didn't see any overlaycallback in the templates

Reply:
Darn yes there is one ok let me check

Reply:
It's only used in the FTLs to check if the Engine is running in preview mode, I'll create a ticket to fix that.